src/app/shared/ui-kit/ui-header/ui-header.component.html

```diff
@@ -1,20 +1,90 @@
-<header
-  class="flex w-full items-center justify-around space-x-8 bg-primary p-4"
->
-  <div>
-    <img src="assets/images/logo.svg" alt="Company Logo" />
+<nav class="relative mx-5 flex items-center justify-between p-4">
+  <div class="flex items-center">
+    <a [routerLink]="''"
+      ><img
+        src="assets/images/logo.svg"
+        alt="Company Logo"
+        class="min-h-[40px] min-w-[74px]"
+    /></a>
   </div>
-  <nav>
-    <ul class="flex space-x-4">
+
+  <div class="mx-40 hidden items-center justify-between space-x-8 md:flex">
+    <ul class="flex space-x-6">
       @for (item of navItems; track $index) {
-        <li>
-          <a [routerLink]="item.link" class="text-bright">{{ item.label }}</a>
+        <li class="duration-100 ease-in md:hover:scale-125">
+          <a [routerLink]="item.link" routerLinkActive="font-semibold">{{
+            item.label
+          }}</a>
         </li>
       }
     </ul>
-  </nav>
-  <div>
-    <quiz-ui-button route="/">Button 1</quiz-ui-button>
-    <quiz-ui-button route="/">Button 2</quiz-ui-button>
   </div>
-</header>
+
+  <div class="hidden items-center space-x-8 md:flex">
+    <quiz-ui-button> Login </quiz-ui-button>
+  </div>
+
+  <!-- Mobile Menu Button -->
+  <button
+    class="z-10 text-white md:hidden"
```

Comment:
Added SVG icons using this library.

I decided to keep the default button because changing it to ui-kit-button would require updating other components as well, and I don’t have enough time for that right now.

Reply:
Ok, add a ticket to the backlog to update the ui kit button and re-use it